core/src/main/java/org/elasticsearch/search/query/QuerySearchRequest.java

```diff
@@ -72,10 +79,7 @@ public IndicesOptions indicesOptions() {
 
     @Override
     public void readFrom(StreamInput in) throws IOException {
-        super.readFrom(in);
-        id = in.readLong();
-        dfs = readAggregatedDfs(in);
-        originalIndices = OriginalIndices.readOriginalIndices(in);
+        throw new UnsupportedOperationException("usage of Streamable is to be replaced by Writeable");
```

Comment:
Can you move `writeTo` to be under the reading ctor when you do this?

Reply:
in general. for all of them? So we have `writeTo` under the last constructor for all these objects?

Reply:
For all of them, yeah. I like having the write and read on close physically so I can look at them at the same time.

Reply:
cool. finished moving all of them